designer/server/src/routes/forms/api.js

```diff
@@ -53,7 +54,10 @@ export default [
             )
           }
 
-          await forms.updateDraftFormDefinition(id, value)
+          const value = result.value
```

Comment:
Maybe we don't need `const value` here either?

But it might be a nice way to show the schema value is a form `definition`?

```suggestion
          const { value: definition } = result
```